neurodamus/metype.py

```diff
@@ -35,6 +36,63 @@ def connect2target(self, target_pp=None):
         """Connects empty cell to target"""
         return Nd.NetCon(self._cellref, target_pp)
 
+    @staticmethod
+    def get_sec(cell, section_id):
+        """Return the section corresponding to the global section_id
+        Note: supports python's negative indices.
+        """
+        # Build a list of (section_list, length) tuples for available section types
+        sec_lists = [getattr(cell, stype) for stype in BaseCell._sections if hasattr(cell, stype)]
+
+        # Compute total length
+        total_len = sum(len(lst) for lst in sec_lists)
+
+        # Handle negative index
```

Comment:
why does it need to handle negative ids?
I find these comments not very helpful; I can see that it's doing something w/ a negative index, but it's not clear *why* it has to handle a negative index.

The `# Compute total length` comment falls in the same boat, imo.

Reply:
It's not about "coming after you"; it's a "feature" that isn't needed, and it's a good guard for to check for misuse of section_ids

Reply:
I do not see it as a big deal since negative indexes are widely used in python. anyway, np. removed